owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/utils/LoggingHelper.kt

```diff
@@ -13,7 +13,7 @@ object LoggingHelper {
         }
         if (!directory.exists())
             directory.mkdirs()
-        Timber.plant(FileLoggingTree(directory, filename = storagePath, delegator = Log_OC::class.java))
+        Timber.plant(FileLoggingTree(directory, filename = storagePath, delegator = Timber::class.java))
```

Comment:
As you were who updated this previously, what do you think about this? @hannesa2

Reply:
Simply ignore the parameter `delegator`

`Timber.plant(FileLoggingTree(directory, filename = storagePath))`

Reply:
Ok, then I merge https://github.com/hannesa2/Logcat/pull/31 and create new version, because it was only a feature for `ownCloud` to be able to delegate

Reply:
Please move now to 
`implementation 'com.github.hannesa2:logcat:1.6.0'`
without delegation